### DISCUSSION TITLE
GLX error on startup - SDR++ with Debian 11 and VNCServer

### DISCUSSION THREAD

**Discussion by jimjackii** [AlexandreRouma/SDRPlusPlus#1213] [General]
Hello,
I have a Debian 11 machine running headless with VNC Server.

I installed SDR++ on it and now want to start SDR++ via VNC server.

Unfortunately SDR++ does not start and the following error messages appear - see screenshot link.

https://jimjackii.de/bilder/sdrpp/debian11_sdrpp_start_error.jpg
(Sorry - how can I include an image in the post?)

It's an eror dependig OpenGL
Glfw Error 65542: GLX: NoGLXFBConfigs returned
OpenGL 3.0 was not supported
...
OpenGL 3.1 ES was not supported
...
OpenGL 2.1 was not supported

I try it also with the last nightly build SDR++ V1.1.0. It occured the same error.

Now I don't know what to do next - which packages are missing or still need to be installed?

**Comment by starvald** [comment]
VNC doesn't have opengl, not completely headless but you can buy a fake monitor dongle then share that screen which will have opengl support.

i've used realvnc to do it this way on a raspberry pi or on a mini pc i've used KDE Plasma's desktop sharing program.

**Comment by jimjackii** [comment]
Ok, Thanks.

I will try this.

**Comment by starvald** [comment]
Plasma desktop sharing app does require a setting to be enabled for unattended use, so you may have to connect a monitor at least once to enable that unless its stored in a config file somewhere.
